docs-ref-conceptual/get-started-tutorial-0-before-you-begin.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Learn about Azure CLI environments, terms, and compare the Azure CLI to Azure PowerShell | Microsoft Docs
+description: Learn the environments you can use to execute Azure CLI commands, Azure CLI terms, how the Azure CLI compares to Azure PowerShell.
+manager: jasongroce
+author: dbradish-microsoft
+ms.author: dbradish
+ms.date: 11/15/2023
+ms.topic: conceptual
+ms.service: azure-cli
+ms.tool: azure-cli
+ms.custom: devx-track-azurecli
+keywords: azure, az account, az group, az config, az init, configuration
+---
+# Welcome to Azure CLI
+
+We're glad you are here! You have chosen a tool that will allow you to create, update, and delete Azure resources from a command-line. In this tutorial, you will onboard with the Azure CLI and learn the following skills that will help you to be successful in your Azure CLI journey:
+
+> [!div class="checklist"]
+>
+> * 1 - Get to know the Azure CLI in this tutorial step
+> * 2 - [Prepare your environment for the Azure CLI](./get-started-tutorial-1-prepare-environment.md)
+> * 3 - [Learn Azure CLI syntax differences in Bash, PowerShell and Cmd](./get-started-tutorial-2-environment-syntax.md)
+> * 4 - [Populate variables for use in scripts](./get-started-tutorial-3-use-variables.md)
+> * 5 - [Delete Azure resources at scale using a script](./get-started-tutorial-4-delete-resources.md)
+
+## What is the Azure CLI?
+
+The Azure Command-Line Interface (CLI) is a cross-platform command-line tool to connect to Azure and execute administrative commands on Azure resources. It allows the execution of commands through a terminal using interactive command-line prompts or a script.
+
+For interactive use, first launch a shell such as Bash, PowerShell, or Cmd.exe on Windows, Linux or macOS.  Then issue a command at the shell prompt. To automate repetitive tasks, assemble the Azure CLI commands into a shell script using the script syntax of your chosen shell, and then execute the script.
+
+The Azure CLI can be installed locally on Linux, macOS, or Windows computers. Azure CLI can also be used from a browser through Azure Cloud Shell or run from inside a Docker container.
+
+## What shell environment should I use?
+
+Windows PowerShell, PowerShell, Cmd, and Bash are shell environments. Your shell environment not only determines which tools you can use, but your shell also changes your command-line experience. For example, there are important quoting differences, line continuation characters, and variable syntax differences between shells.
```

Comment:
Suggestion "Windows PowerShell, PowerShell, Cmd, and Bash are different shell environments."